app/assets/stylesheets/application.scss

```diff
@@ -1,16 +1,5 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require devise_bootstrap_views
- *= require_self
- */
+@import "devise_bootstrap_views";
+
+.navbar {
```

Comment:
I suggest you store that style in the `app/assets/stylesheets/navbar.scss` and then import that file in the `app/assets/stylesheets/application.scss`